openfl-workspace/flower-app-pytorch/plan/plan.yaml

```diff
@@ -52,12 +52,12 @@ assigner :
       - name  : Connector_Flower
         percentage : 1.0
         tasks : 
-          - start_client_adapter
+          - interop
```

Comment:
I'd suggest the more explicit `prepare_for_interop` (inspired by the related python method name)